performance-test/tasks/notification_tasks.py

```
@@ -1,15 +1,54 @@
-        params = {"limit": 50}
-        self.client.get("/api/notifications",
-                        headers=self.parent.get_headers(),
-                        params=params)
+    def on_start(self):
+        # 이미 정의되어 있음
+        pass
+
+    def pick_new_user(self):
+        """알림이 없는 유저를 만났을 때 호출하여 다른 유저 ID로 교체"""
+        from locustfile import user_id_pool # locustfile에 있는 풀을 참조
+        self.user_id = random.choice(user_id_pool)
```

Ruff (0.15.12)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)